Sources/Sentry/Profiling/SentryThreadMetadataCache.cpp

```diff
@@ -0,0 +1,58 @@
+#include "SentryThreadMetadataCache.hpp"
+
+#include "SentryThreadHandle.hpp"
+#include "SentryStackBounds.hpp"
+
+#include <algorithm>
+#include <string>
+#include <vector>
+
+namespace {
+
+bool isSentryOwnedThreadName(const std::string &name) {
+    return name.rfind("io.sentry", 0) == 0;
```

Comment:
We could, or I could change mine to match the existing format to avoid renaming a bunch of other stuff -- do you know if all threads created by the SDK use the `Sentry` prefix?